### PR TITLE
Create a schema using the project.dbName instead of the id

### PR DESCRIPTION
**Pull request by do4gr**
Fixes https://github.com/prisma/prisma/issues/4086 https://github.com/prisma/prisma/issues/4076

### CODE REVIEW

File: server/servers/deploy/src/test/scala/com/prisma/shared/schema_dsl/SchemaDsl.scala [hunk 1]
```diff
@@ -77,7 +78,10 @@ object SchemaDsl extends AwaitUtils {
 
     val schema                 = SchemaInferrer(deployConnector.capabilities).infer(emptyBaseSchema, emptySchemaMapping, prismaSdl, inferredTables)
     val withBackRelationsAdded = MissingBackRelations.add(schema)
-    val manifestation          = ProjectManifestation(database = Some(id + "_DB"), schema = Some(id + "_S"))
+    val manifestation = ConfigLoader.load().databases.head.connector match {
+      case x if x == "postgres" => ProjectManifestation(database = Some(id + "_DB"), schema = Some(id + "_S"), x)
+      case y                    => ProjectManifestation(database = Some(id + "_S"), schema = None, y)
```
Comment (mavilein) [inline server/servers/deploy/src/test/scala/com/prisma/shared/schema_dsl/SchemaDsl.scala]: Shouldn't that be `_DB` like above?

File: server/shared-models/src/main/scala/com/prisma/shared/models/Project.scala [hunk 2]
```diff
@@ -20,14 +20,18 @@ case class Project(
   val serverSideSubscriptionFunctions = functions.collect { case x: ServerSideSubscriptionFunction => x }
 
   val dbName: String = manifestation match {
-    case ProjectManifestation(Some(_), Some(schema)) => schema // Postgres Passive
-    case ProjectManifestation(Some(database), None)  => database // Mongo + MySQL Passive
-    case ProjectManifestation(None, Some(_))         => sys.error("The configloader should have rejected this.") // Invalid
-    case ProjectManifestation(None, None)            => id // All active
+    case ProjectManifestation(Some(_), Some(schema), "postgres")                  => schema
+    case ProjectManifestation(Some(_), Some(schema), _)                           => sys.error("The configloader should have rejected this.")
```
Comment (mavilein) [inline server/shared-models/src/main/scala/com/prisma/shared/models/Project.scala]: Can you add explanations why those cases that error are invalid? I can't remember anymore :D

File: server/servers/deploy/src/test/scala/com/prisma/shared/schema_dsl/SchemaDsl.scala (hunk 1 above)
Comment (do4gr) [reply server/servers/deploy/src/test/scala/com/prisma/shared/schema_dsl/SchemaDsl.scala]: You're right.

File: server/shared-models/src/main/scala/com/prisma/shared/models/Project.scala (hunk 2 above)
Comment (do4gr) [reply server/shared-models/src/main/scala/com/prisma/shared/models/Project.scala]: Done

**Pull request merged**